File: py-c/fineshed assighnments/functions paractice/updateBudget.py

print("this is your new finacial calculator")

def usrinpt(type):
    return f"{type}"

income = float (usrinpt(input("whats your monthly income: ")))
rent = float (usrinpt(input("whats your monthly rent: ")))
utilitys = float (usrinpt(input("whats your monthly utilities: ")))
groseries = float (usrinpt(input("whats your monthly groseries: ")))
transportation = float (usrinpt(input("whats your monthly transportation: ")))
expenses = rent+utilitys+groseries+transportation
savings = income*.2
total = income-expenses-savings
print("your income is", income)
print("here are your savings", savings)
print("here are your expenses", expenses)
print("heres what you have left to spend", total)

def percent(type,amount):
    per = amount/income*100
    return f"{type} is {per}% of your income"

print(percent("rent",rent))
print(percent("utillities",utilitys))
print(percent("grosseries",groseries))
print(percent("transportation",transportation))

File: py-c/fineshed assighnments/functions paractice/updateBudget.c
#include <stdio.h>
float income, rent, groseries, transportation, utillities, expenses, savings, total;

float input(char type[],float var){
    printf("how much is your %s: \n", type);
    scanf("%f",&var);
}

void percent(char type[],int amount){
    int per = amount/income*100;
    printf("your percent %s is %d%% of your income\n",type, per);
}

int main(void){
    printf("this is your new finacial calculator");
    income = input("income",income);
    rent = input("rent",rent);
    groseries = input("grosseries",groseries);
    transportation = input("transportation",transportation);
    utillities = input("utilities",utillities);
    expenses = rent + groseries + transportation +utillities;
    savings = income*0.2;
    total = income - (expenses + savings);
    printf("your income is: %.2f\n",income);
    printf("here are your savings %.2f\n", savings);
    printf("here are your expences %.2f\n", expenses);
    printf("heres what you have left to spend %.2f\n", total);
    percent("rent",rent);
    percent("utillities",utillities);
    percent("groseries",groseries);
    percent("transeportation",transportation);
    return 0;
}